main.go

```diff
@@ -38,10 +38,10 @@ var (
 
 var (
 	enableDeprecatedMetrics = true // default may change in future
-	deprecatedMetrics       = kingpin.Flag("metrics.deprecated", "Enable or disable deprecated metrics (`ping_rtt_ms{type=best|worst|mean|std_dev}`). Valid choices: [enable, disable]").Default("enable").String()
+	deprecatedMetrics       = kingpin.Flag("metrics.deprecated", "Enable or disable deprecated metrics (`ping_rtt_ms{type=best|worst|mean|std_dev}`). Valid choices: [enable, disable]").Default("disable").String()
 
 	rttMetricsScale = rttInMills // might change in future
-	rttMode         = kingpin.Flag("metrics.rttunit", "Export ping results as either millis (default), or seconds (best practice), or both (for migrations). Valid choices: [ms, s, both]").Default("ms").String()
+	rttMode         = kingpin.Flag("metrics.rttunit", "Export ping results as either seconds (default), or milliseconds (deprecated), or both (for migrations). Valid choices: [ms, s, both]").Default("s").String()
```

Comment:
Very small nitpick: `Valid choices: [ms, s, both]")` should order the values as `[s, ms, both]` to match the order they are introduced in the text, and moves the default to the front (as is most common).

Reply:
Fixed!